README.md

```diff
@@ -112,6 +112,14 @@ updating your `ember-cli` and/or `ember-cli-qunit`/`ember-cli-mocha`
 dependencies.
 
 
+### With Addons
+
+Please note that if you are using this to lint an addon, files which are part
+of the build process (ie. index.js, ember-cli-build.js, config/) will not be linted. It
```

Comment:
this is actually true for apps as well. e.g. the `ember-cli-build.js` of an app will also not be linted by this addon.